hello-world-apb/Makefile

```diff
@@ -30,11 +30,9 @@ openshift-test:
 	docker login -u ${OC_USER} -p ${OC_PASS} ${REGISTRY}:5000
 	docker tag ${CONTEXT}/${IMAGE_NAME}:${TARGET}-${VERSION} ${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME}
 	docker push ${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME}
-	oc adm policy add-role-to-user admin -z default
-	oc run ${IMAGE_NAME} --restart=Never --image=${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME} -- provision --extra-vars 'namespace=${PROJ_RANDOM}'
-	while [ -z $${POD_STATE} ]; do sleep 2; POD_STATE=$$(oc get pod ${IMAGE_NAME} --template '{{(index .status.containerStatuses 0).state.terminated.exitCode}}' | grep -w '^[0-9]*$$'); done
-	oc describe po/${IMAGE_NAME}
-	oc logs po/${IMAGE_NAME}
+	oc create sa apb
+	oc adm policy add-role-to-user admin -z apb
+	oc run ${IMAGE_NAME} --image=${REGISTRY}:5000/${PROJ_RANDOM}/${IMAGE_NAME} --restart=Never --attach=true --overrides='{"apiVersion":"v1","spec":{"serviceAccountName":"apb"}}' -- provision -e namespace=${PROJ_RANDOM}
```

Comment:
+1 nice to see how we can specify the service account name with oc run....very cool!  Need to get this in a FAQ so others are aware.